### ISSUE TITLE
Brand slides as Apache 2

### ISSUE THREAD

**Issue by sergioarmgpl** [sergioarmgpl/operating-systems-usac-course#9]
This includes right know a Kubernetes GCP introduction, it could include technologies as containers, linkerd, etc

**Comment by Jose-Carlos-Jimenez** [comment]
Hi @sergioarmgpl , i would like to work in this issue with a teammate, his github user is @ferjotello.

**Comment by FerJoTello** [comment]
Yes, I would like to work in the issue with @Jose-Carlos-Jimenez

**Comment by cloudsociety-dev** [comment]
Let me share you the slides https://docs.google.com/presentation/d/1ZBBPHlI3CBUaukW1nyIn_xiygyqavjgGytAR_daUmdk/edit?usp=sharing please ask for edition permissions

**Comment by sergioarmgpl** [comment]
Let me share you the slides https://docs.google.com/presentation/d/1ZBBPHlI3CBUaukW1nyIn_xiygyqavjgGytAR_daUmdk/edit?usp=sharing please request editor permissions
